app/models/katello/glue/candlepin/owner.rb

```diff
@@ -21,7 +21,9 @@ def owner_info
       end
 
       def owner_details
```

Comment:
I thought about it, and it seemed stranger to put on the organization Rabl itself. The proper solution seemed to have a 'manifest' rabl and API since we talk about it as a first class object but that seemed larger in scope.